ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp

```diff
@@ -1,4 +1,4 @@
-/** Copyright (C) 2019 Samuel Venable
+/** Copyright (C) 2018-2019 Samuel Venable
 ***
 *** This file is a part of the ENIGMA Development Environment.
 ***

```

Comment:
For future reference, if you make a commit you don't want built on the CI, you can put `[skip ci]` in your commit title. I will allow this for comments and simple changes where there is a previous commit that already passed the CI. This commit here is a perfect example of where I would have found it acceptable. Yes, this applies to commits made from GitHub UI as well as from git on the command line.

Reply:
Will do for comment edits here on out (btw in regards to me not "ever listening" to you it is not on purpose. I'm not as big a geek and my life gets wrapped up in other things in life so I often forget shit like this when not used often